test/uk/gov/hmrc/plasticpackagingtax/registration/views/models/ListGroupMembersViewModelSpec.scala

```diff
@@ -64,5 +64,28 @@ class ListGroupMembersViewModelSpec extends PlaySpec with PptTestData {
           other.subHeading mustBe None
       }
     }
+
+    "not contain remove button" when {
+      "has two member only" in {
+        val newSut = createSut(createRegistration(Seq(groupMember)))
+
+        newSut.listMembers(mockMessages).foreach(_.remove mustBe None)
+      }
+
+      "has one member only" in {
+        val newSut = createSut(createRegistration(Seq.empty))
+
+        newSut.listMembers(mockMessages).foreach(_.remove mustBe None)
+      }
+    }
+  }
+
+  private def createRegistration(members: Seq[GroupMember]): Registration = {
+    val groupDetail1 = GroupDetail(membersUnderGroupControl = Some(true), members = members)
+    aRegistration(withGroupDetail(Some(groupDetail1)))
   }
+
+  private def createSut(registration: Registration): ListGroupMembersViewModel =
```

Comment:
Non blocking comment - what's a Sut?
If this is an abbreviation could it have been written out in full; because HMRC already has many acronyms.